### DISCUSSION TITLE
Can't get to playback the recording in browser

### DISCUSSION THREAD

**Discussion by Abdulee** [blakeblackshear/frigate#9654] [Ask A Question]
Hello,
I have just setup my camera in frigate(it is a pretty generic 25€ IP Cam) and I am able to get live playback in frigate web interface. However after setting up recordings I can't play the recording in my browser and am presented with: Playback cannot continue. No available working or supported playlists.
I can just download the video and view it fine but in frigate I can't view it. I've read in the FAQ: and it says that I need to ensure that the camera's sending H264 encoded video or transcode it.. I think it might just be that my camera is sending H265 encoded video. How can I transcode this. Thanks a lot.

**Comment by NickM-27** [comment]
What browser are you using

**Reply by Abdulee** [reply]
brave(based on chromium) I've also tried firefox but same result..

**Reply by NickM-27** [reply]
I don't believe either of those support h265. I'd try chrome

**Reply by Abdulee** [reply]
I believe they do.. Either way I've tried chrome with no success either: same message as before

**Reply by NickM-27** [reply]
Firefox does not support h265. In any case you will want to use media internals on chrome to see why it's failing

**Reply by marleyjaffe** [reply]
Chrome solved the issue for me. Is there any way to restream to H264 on playback (either always or when viewing from an unsupported browser)?

**Reply by NickM-27** [reply]
No, on demand transcoding is not supported

**Reply by marleyjaffe** [reply]
So in order to get this to work on Mac Firefox (seems like there is some experimental h265 firefox support on windows), I would need to change my cameras to H264 instead, right? The primary impact of doing so is that the storage footprint will be larger. Are both those points correct? Thanks for the quick response!

**Reply by NickM-27** [reply]
yes, if you are tied to firefox. Safari, Chrome, and I believe some others support h265

**Reply by marleyjaffe** [reply]
OK, Makes sense. The primary impact of h264 swap will be a larger event clip / recording file size due to the different compression, right?

**Reply by NickM-27** [reply]
yes
